apps/web/components/connected/UmbodsmadurSkuldara/UmsCostOfLivingCalculator.tsx

```diff
@@ -562,13 +568,25 @@ const UmsCostOfLivingCalculator = ({ slice }: CostOfLivingCalculatorProps) => {
                 </Stack>
               </Box>
             </Stack>
-
-            <Text variant="default" fontWeight="semiBold">
-              {n('totalText', activeLocale === 'is' ? 'Samtals' : 'Total')}
-              {': '}
-              {numberFormatter.format(getTotal())}{' '}
-              {n('currencyText', activeLocale === 'is' ? 'krónur' : 'ISK')}
-            </Text>
+            <Stack space={2}>
+              <Text variant="h3" fontWeight="semiBold">
+                {n('totalText', activeLocale === 'is' ? 'Samtals' : 'Total')}
+              </Text>
+              <Box className={styles.total}>
+                <Text variant="eyebrow" fontWeight="semiBold" color="blue400">
+                  {n('totalText', activeLocale === 'is' ? 'Samtals' : 'Total')}
+                </Text>
+                <Box paddingLeft={2} textAlign={'right'}>
```

Comment:
We could have this as textAlign="right" and that way there is no need for the curly bracket